utils/data_utils.py

```diff
@@ -0,0 +1,80 @@
+"""
+Data processing utils
+"""
+
+
+def process_tissue_ccle_column(
+    input_df,
+    recode_col_name="CCLE Cell Line Name",
+    nan_col_to_fill="cell_line_name",
+    missing_tissue_dict={},
+):
+    """
+    Split a cell line id column into tissue and cell line name, thus cleaning
+
+    Arguments:
+    ----------
+    input_df : pandas.DataFrame
+        The data frame containing the column to clean
+    recode_col_name : str, default "CCLE Cell Line Name"
+        The specific column to clean
+    nan_col_to_fill : str, default "cell_line_name"
+        Which column to fill missing values, if CCLE name does not exist
+    missing_tissue_dict : dict, default {}
+        Optional argument to fill in missing details
+
+    Returns:
+    --------
+        The same input dataframe with a cleaned tissue and cell line column
+    """
+    cell_line_part = input_df.loc[:, recode_col_name].apply(
```

Comment:
Depending on Pandas implementation preference, would it make sense to simplify as follows?
```suggestion
    cell_line_part = input_df[recode_col_name].apply(
```

Reply:
IMO `.iloc` has higher readability than the bracket - I like to confirm that I am specifically accessing a column